applications/nrf_desktop/configuration/nrf52kbd_nrf52832/prj.conf

```diff
@@ -184,22 +185,13 @@ CONFIG_NFCT_PINS_AS_GPIOS=y
 
 CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
+CONFIG_ASSERT_VERBOSE=n
 
-CONFIG_LOG=y
-CONFIG_LOG_DEFAULT_LEVEL=2
-CONFIG_LOG_BACKEND_RTT=y
-CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
-CONFIG_LOG_MODE_OVERFLOW=y
-CONFIG_LOG_PRINTK=y
-CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=256
-CONFIG_LOG_BUFFER_SIZE=4096
-CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE=256
-CONFIG_LOG_BACKEND_SHOW_COLOR=n
-CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=n
-CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
-
-CONFIG_CONSOLE=y
-CONFIG_USE_SEGGER_RTT=y
-CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
-CONFIG_RTT_CONSOLE=y
-CONFIG_UART_CONSOLE=n
+################################################################################
```

Comment:
this one is tricky - we won't have configuration with logs... let's talk about it offline